mat/vector.go

```diff
@@ -23,7 +23,7 @@ type Vector struct {
 	mat blas64.Vector
 	n   int
 	// A BLAS vector can have a negative increment, but allowing this
-	// in the mat64 type complicates a lot of code, and doesn't gain anything.
+	// in the mat type complicates a lot of code, and doesn't gain anything.
```

Comment:
perhaps name it `mat.Vector` otherwise it could be (a bit) confusing with the `mat` field, above ?